include/zenoh-pico/api/constants.h

```diff
@@ -133,24 +135,24 @@ typedef enum
  */
 typedef enum
 {
-    Z_QUERYABLE_ALL_KINDS = 0x01,  // 1 << 0
-    Z_QUERYABLE_STORAGE = 0x02,    // 1 << 1
-    Z_QUERYABLE_EVAL = 0x04        // 1 << 2
+    Z_QUERYABLE_ALL_KINDS = 0x01, // 1 << 0
+    Z_QUERYABLE_STORAGE = 0x02,   // 1 << 1
+    Z_QUERYABLE_EVAL = 0x04       // 1 << 2
 } z_queryable_kind_t;
 
 /**
  * Consolidation mode values.
  *
  * Enumerators:
- *     Z_CONSOLIDATION_MODE_FULL: Guaranties unicity of replies. Optimizes bandwidth.
- *     Z_CONSOLIDATION_MODE_LAZY: Does not garanty unicity. Optimizes latency.
+ *     Z_CONSOLIDATION_MODE_LAST_VALUE: Guaranties unicity of replies. Optimizes bandwidth.
+ *     Z_CONSOLIDATION_MODE_MONOTONIC: Does not garanty unicity. Optimizes latency.
```

Comment:
I suggest to have the order in the comments to be the same as in the enum definition.

Reply:
Fixed and improved docs.